### PR TITLE
feat(backupstore): add azurite emulator

### PR DESCRIPTION
**Pull request by mantissahz**
Add the Azurite manifest to start Azurite server for local testing

Ref: longhorn/longhorn#1309, https://github.com/longhorn/longhorn/issues/5994

### CODE REVIEW

**Comment by innobead** [comment]
@mantissahz Can we add to longhorn/longhorn as well together with minio & nfs manifests for backupstore? Thanks.

**Review by chriscchien — APPROVED** [review]

**Comment by mantissahz** [comment]
Added to longhorn/longhorn as well together with minio & nfs manifests for backupstore in this PR https://github.com/longhorn/longhorn/pull/6006

File: manager/integration/deploy/backupstores/azurite-backupstore.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+data:
+  AZBLOB_ACCOUNT_NAME: ZGV2c3RvcmVhY2NvdW50MQ==
+  AZBLOB_ACCOUNT_KEY: RWJ5OHZkTTAyeE5PY3FGbHFVd0pQTGxtRXRsQ0RYSjFPVXpGVDUwdVNSWjZJRnN1RnEyVVZFckN6NEk2dHEvSzFTWkZQVE90ci9LQkhCZWtzb0dNR3c9PQ==
+  AZBLOB_ENDPOINT: aHR0cDovL2F6YmxvYi1zZXJ2aWNlLmRlZmF1bHQ6MTAwMDAv
+---
+# same secret for longhorn-system namespace
+apiVersion: v1
+kind: Secret
+metadata:
+  name: azblob-secret
```
Comment (innobead) [inline manager/integration/deploy/backupstores/azurite-backupstore.yaml:16]: Why having two azblob-secret in different namespaces?

**Review by innobead — CHANGES_REQUESTED** [review]
Let's close this PR and create another issue to leverage the backup store deployment script from longhorn/longhorn instead of another duplicated manifest here.

**Comment by mantissahz** [comment]
Create an issue to follow up, We need to use manifests from longhorn/longhorn

**Pull request closed without merging**